### PR TITLE
Document preconfiguring QUnit

### LINKED ISSUE #140: Document predefining QUnit.config
Ref: https://github.com/qunitjs/qunit/pull/1060

### PR DESCRIPTION
**Pull request by TobiasOetzel**
closes #140

### CODE REVIEW

File: entries/QUnit.config.xml [hunk 1]
```diff
@@ -168,5 +168,23 @@ QUnit.config.urlConfig.push({
 });
 ]]></code>
 	</example>
+	<example>
+		<desc>
+			Preconfiguring QUnit
```
Comment (trentmwillis) [inline entries/QUnit.config.xml]: For consistency, let's wrap this in a `<p>` element.

File: entries/QUnit.config.xml [hunk 2]
```diff
@@ -168,5 +168,23 @@ QUnit.config.urlConfig.push({
...
+			<p>
+				If you want to configure QUnit before it is loaded, it is allowed to introduce the global variable <code>QUnit</code> with the property <code>config</code>.
```
Comment (trentmwillis) [inline entries/QUnit.config.xml]: Can we reword this slightly to:
```
If you want to configure QUnit before it is loaded, you can introduce the global variable <code>QUnit</code> with the property <code>config</code> specified.
```

File: entries/QUnit.config.xml [hunk 3]
```diff
@@ -168,5 +168,23 @@ QUnit.config.urlConfig.push({
...
+				All other properties of the QUnit object will be ignored. In the config properties you may specify any of the allowed QUnit.config values.
```
Comment (trentmwillis) [inline entries/QUnit.config.xml]: Let's keep this on the same line as the last sentence. Similar to how the rest of the document is setup.

**Comment by trentmwillis** [comment]
@TobiasOetzel are you able to update this? If not, I can make the changes myself, but want to give you a chance to carry it through.

**Comment by TobiasOetzel** [comment]
Sorry i forgot this one

**Pull request merged**